be/src/olap/merger.cpp

```diff
@@ -207,16 +207,25 @@ void Merger::vertical_split_columns(const TabletSchema& tablet_schema,
         column_groups->emplace_back(std::move(key_columns));
     }
     auto&& cluster_key_idxes = tablet_schema.cluster_key_idxes();
+
+    std::vector<uint32_t> value_columns;
     for (uint32_t i = num_key_cols; i < total_cols; ++i) {
         if (i == sequence_col_idx || i == delete_sign_idx ||
             cluster_key_idxes.end() !=
                     std::find(cluster_key_idxes.begin(), cluster_key_idxes.end(), i)) {
             continue;
         }
-        if ((i - num_key_cols) % config::vertical_compaction_num_columns_per_group == 0) {
-            column_groups->emplace_back();
+
+        if (!value_columns.empty() &&
```

Comment:
For the unqiue key table,if there are only key columns in the schema, why does the code run here ? After all, besides the key column, there are only delete sign and sequence columns, which are skipped at line 213.